README.md

```diff
@@ -48,41 +48,52 @@ c2patool image.jpg -d
 
 ### Adding a manifest to a file
 
-You can add C2PA data to a file by passing a [manifest definition](#manifest-definition-format) JSON file instead of an image. The tool will generate a new manifest using the values given in the definition. 
+You can add C2PA data to a file by passing a [manifest definition](#manifest-definition-format) JSON file using the -m manifest option. The tool will generate a new manifest using the values given in the definition. 
```

Comment:
@crandmck would you review these changes for clarity?

Reply:
Suggest rewording as follows:

To add C2PA data to a file, use the `-m manifest` option and provide a [manifest definition](#manifest-definition-format) JSON file. The tool generates a new manifest using the values given in the definition.